R/customVarInfo.R

```diff
@@ -157,7 +157,7 @@ getCustomInfo <- function(v, info, default = NULL, onError = NULL) {
 #' @export
 .vsc.getAllVarInfos <- function(){
   varInfos <- session$varInfos
-  varInfos <- lapply(seq(1, length(varInfos)), function(i){
+  varInfos <- lapply(seq_len(length(varInfos)), function(i){
```

Comment:
You might also want `seq_along(x)` in this particular case, equilvalent to `seq_len(length(x))`